views/layouts/main.handlebars

```diff
@@ -52,6 +52,11 @@
           </div>
           <br>
 
+	<div id="overlay">	
+	<div class="spinner"></div>
+</div>
+	
+	<div id="realContent" style="visibility:hidden">
           <h3>{{msg}}</h3>
           <div class="result" style="display:{{showResult}}">
```

Comment:
One more thing, I thought we already deleted those lines? Please refer to the latest `main.handlebars`